spec/factories/payment_responses.rb

```diff
@@ -0,0 +1,86 @@
+FactoryBot.define do
+  sequence :adyen_payments_response do
+    OpenStruct.new(
+      response: {
+        "additionalData" => {
+            "recurringProcessingModel" => "UnscheduledCardOnFile"
+        },
+        "pspReference" => "#{SecureRandom.uuid}",
+        "resultCode" => "Authorised",
+        "amount" => {
+            "currency" => "USD",
+            "value" => 1000
```

Comment:
Style/TrailingCommaInHashLiteral: Put a comma after the last item of a multiline hash.